etor/src/main/java/gov/hhs/cdc/trustedintermediary/external/hapi/HapiOrderConverter.java

```diff
@@ -105,6 +106,40 @@ public Order<?> convertMetadataToOmlOrder(Order<?> order) {
         return new HapiOrder(orderBundle);
     }
 
+    @Override
+    public Order<?> addContactSectionToPatientResource(Order<?> order) {
+        logger.logInfo("Adding contact section in Patient resource");
+
+        var hapiOrder = (Order<Bundle>) order;
+        var orderBundle = hapiOrder.getUnderlyingOrder();
+        var codingList = new ArrayList<Coding>();
+        codingList.add(
+                new Coding()
+                        .setSystem("http://terminology.hl7.org/CodeSystem/v3-RoleCode")
+                        .setCode("MTH")
+                        .setDisplay("mother"));
```

Comment:
This seems to be static and doesn't change.  Let's move this up to the top of this class similar to the `OML_CODING` field.

Also, you can use `List.of(...)` and a single constructor.  For example...

```java
List.of(new Coding("http://terminology.hl7.org/CodeSystem/v3-RoleCode", "MTH", "mother"))
```

Reply:
done.